containers/keycloak-bootstrap/keycloak_bootstrap.py

```diff
@@ -187,12 +191,14 @@ def createTestClientForX509Flow(keycloak_admin):
                 "clientId": client_id,
                 "publicClient": "false",
                 "standardFlowEnabled": "true",
+                "directAccessGrantsEnabled": "true",
+                "serviceAccountsEnabled": "true",
                 "clientAuthenticatorType": "client-x509",
                 "baseUrl": "https://local.virtru.com/",
                 "protocol": "openid-connect",
                 "redirectUris": ["https://local.virtru.com/*"],
                 "webOrigins": ["+"],
-                "attributes": {"x509.subjectdn": "CN=(.*)(?:$)"},
+                "attributes": {"x509.subjectdn": "CN=john"},
```

Comment:
```suggestion
                "attributes": {"x509.subjectdn": "CN=john"},
```

This can't be hardcoded to a specific user, because we (DCR/h3) use this X509 test client with other users (and nonbrowser flows)